packages/fluentui/react-northstar/src/components/Breadcrumb/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import {
+  compose,
+  useFluentContext,
+  useTelemetry,
+  useAccessibility,
+  useStyles,
+  useUnhandledProps,
+  getElementType,
+  childrenExist,
+} from '@fluentui/react-bindings';
+import {
+  commonPropTypes,
+  UIComponentProps,
+  ContentComponentProps,
+  ChildrenComponentProps,
+  SizeValue,
+} from '../../utils';
+import { useBreadcrumbContext } from './breadcrumbContext';
+import { Accessibility } from '@fluentui/accessibility';
+
+export interface BreadcrumbLinkProps
+  extends UIComponentProps<BreadcrumbLinkProps>,
+    ContentComponentProps,
+    ChildrenComponentProps {
+  /** Accessibility behavior if overridden by the user. */
+  accessibility?: Accessibility<never>;
```

Comment:
Please I think we would need accessibility for handling `disabled` props, WDYT?

Reply:
You are right, just did. :)